DirectEmployers/MyJobs
# [PD 2882] date range overhaul

Task was to make the Date Global across all tabs and also carry over to switching over to a new dimension as well.

To Test:

Pull down the repository. You can change the date range and see that it changes for all tabs when switched. The tabs will also have to hit the API again once the date has been changed so you will see that as well.
